lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
```

Comment:
I'm not actually sure–I was simply plumbing messaging through code that already existed. I abstracted the code both you and Wilco had worked on into a separate function to make it easier to follow, here's the commit where it originated: https://github.com/dequelabs/axe-core/commit/7252b57c74e0a3d1d8b731a4920756f731e961b1#diff-4d3897aa0085003d18376b4bb785364aR79

I could try removing it and seeing how it impacts real world scans.